### PR TITLE
re-themed site using fcc colors... again

### PR DESCRIPTION
**Pull request by no-stack-dub-sack**
Adapted logo from FCC Landing page with QL's permission, and re-themed site. Some views:

image

image

image

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -22,6 +22,7 @@
     "redux": "^3.6.0",
     "redux-devtools-extension": "^2.13.0",
     "redux-thunk": "^2.2.0",
+    "semantic-ui": "^2.2.10",
```
Comment (bonham000) [inline client/package.json]: We can remove this now right?

File: client/semantic.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      "compressed": "dist/components/",
+      "themes": "dist/themes/"
+    },
+    "clean": "dist/"
+  },
+  "permission": false,
+  "autoInstall": false,
+  "rtl": false,
+  "version": "2.2.10"
+}
```
Comment (bonham000) [inline client/semantic.json]: Do we still need this file?

**Comment by no-stack-dub-sack** [comment]
@bonham000 I suppose we do not. I can update the PR. hang tight

**Comment by bonham000** [comment]
@no-stack-dub-sack I think `semantic` directory should be moved into the `public` folder. This is a static asset, if we're going to include it like this I don't think it should be included in `client` which is essentially all the core application code. No need for this to be watched by webpack and crowd other code in there when it will never be modified. Because we are building with CRA and we have the ability to include assets which it will build for us, I think we should move the `semantic` directory into `public` and reference it as follows:

`<link rel="stylesheet" href="%PUBLIC_URL%/semantic/dist/semantic.min.css">`

This should still give us a good bundle in the end, I just built this locally with this change and it seems OK. It looks like it will just maintain these assets as is. Also, this is in fact how the actual CDN serves these assets, I believe.

Then, any changes to the color scheme, once made and built elsewhere, will just need to be copied into `semantic.min.css` here.

I think this is a decent solution for including this framework CSS.

**Comment by no-stack-dub-sack** [comment]
@bonham000 Ok, that makes sense. I had thought about putting it in the public folder at some point, but wasn't sure the reason why. Let me make this change on my end and just update this PR again.

**Comment by bonham000** [comment]
@no-stack-dub-sack also, finally, if we are going to commit to adding this color scheme change, shouldn't we rename `teal` to `fccGreen` or `fcc` or something other than `teal`?

Other than that this looks pretty good to me know.

**Comment by no-stack-dub-sack** [comment]
@bonham000 hmmm... yes. I was thinking about that too, and it's possible (in fact, I just realized it was possible), but it will not be ideal. The only way to get semantic to automatically recreate alternate shades of a color is too use their predefined color names. But, they also offer a primaryColor option. And if I am not mistaken, I already have that set up to be the main green color, so we could replace every instance of `teal` with `primaryColor` and that should work. There are some other color reassignments that don't line up perfectly, but we probably won't end up using them (I reassigned all of semantics colors to fcc's color scheme - check out `semantic/src/site/globals/site.variables`).

**Comment by bonham000** [comment]
@no-stack-dub-sack ok, are we using `green`? We could just make `green` the color.

**Comment by no-stack-dub-sack** [comment]
@bonham000 Yeah, we could, there are two shades of green, and I just used the lighter one as teal. Weirdly, the primary thing is only working in some places, so we will have to do that. But then where we want to use the darker green, we will have to use teal. Not sure why the primary thing isn't working. Maybe it only works on buttons. 

This will have to be done later though, not in this PR, because I already deleted all the semantic build stuff. I will do this before it actually gets released in finishing touches stage. For now, just know teal is really green

**Pull request merged**